style-guide.adoc

```diff
@@ -0,0 +1,11 @@
+= Style Guide
+
+This document is a content style guide for content in the `architecture/` directory.
+
+=== Architecture
+
+Content in `architecture/` should be consistent with the https://github.com/Mach30/m30ml[Mach30 modeling language (m30ml)].
+YAML files specified in m30ml should subside in the `0-References/`, `1-StakeholderNeeds`, `2-UserStories`, or `3-Datastructures` directory,
+and be prependend with a number, followed by a dash, the content name in https://wiki.c2.com/?PascalCase[Pascal Case], and a `.yaml`,
+e.g., `2-CubeSatDesignSpecificationRev13.yaml`.
```

Comment:
is this comment copied from somewhere else?

Reply:
> When I check the build output, I see "As a Clinician I want to [object Object] so that I can maintain matching protocols per patient volume", that's worded weirdly. The clinician would want to have confidence that the patient PIP and PEEP will not change over time without adjustment by someone involved in the patient's clinical care, and such adjustments will likely need to be noted on the patient's chart.

What is the activity that the clinician is performing to ensure consistent PEEP or PIP?

Reply:
Sorry. This was context-specific boilerplate from another project. I'll replace it with something relevant to tetra